app/javascript/gobierto_data/webapp/components/sets/VisualizationsItem.vue

```diff
@@ -260,14 +264,18 @@ export default {
         name: vizName
       } = value;
       this.labelValue = vizName
-      this.$root.$emit('updateVizName')
+      if (!this.registrationDisabled || this.isUserLogged) {
```

Comment:
This condition `(!this.registrationDisabled || this.isUserLogged)` is used four times in this file. Use a computed property